### ISSUE TITLE
style: subtle color and size adjustment for <details> in the readme

### ISSUE THREAD

**Issue by ayo-run** [npmx-dev/npmx.dev#2391]
### Describe the feature

There should be a subtle UI difference in the content of a `<details>` element to indicate that the information is "Something small enough to escape casual notice." [reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/details#try_it)

The default on `npmjs.com` has a subtle difference in color and size. See content of "Additional info on vite" [on npmjs.com](https://www.npmjs.com/package/@ayo-run/mnswpr#:~:text=Additional%20info%20on%20Vite%2E%2E%2E), and compare it to when [on npmx.dev](https://npmx.dev/package/@ayo-run/mnswpr#:~:text=Additional%20info%20on%20Vite%2E%2E%2E)

### on npmjs.com

<img width="500" height="390" alt="Image" src="https://github.com/user-attachments/assets/dbc6e69d-aad5-4c93-9145-09a037bd3197" />

### on npmx.dev

<img width="500" height="390" alt="Image" src="https://github.com/user-attachments/assets/e21a7686-5dd3-425b-9b99-ba4341302383" />

### Additional information

- [x] Would you be willing to help implement this feature?

### Final checks

- [x] Read the [contribution guide](https://github.com/npmx-dev/npmx.dev/blob/main/CONTRIBUTING.md).
- [x] Check existing [issues](https://github.com/npmx-dev/npmx.dev/issues).
